### PR TITLE
Feat home page

### PR DESCRIPTION
**Pull request by natali-abyss**

### CODE REVIEW

File: app/app.component.spec.ts [hunk 1]
```diff
@@ -2,9 +2,9 @@ import {
   it,
   inject,
   beforeEachProviders
-} from 'angular2/testing';
+} from '../../../node_modules/angular2/testing.d';
 
-import {AppComponent} from './app.component';
+import {AppComponent} from './app.component.ts';
```
Comment (dydek) [inline app/app.component.spec.ts]: you don't have to add the `ts` suffix explicit , webpack can resolve it

File: app/app.component.spec.ts [hunk 2]
```diff
@@ -2,9 +2,9 @@ import {
   it,
   inject,
   beforeEachProviders
-} from 'angular2/testing';
+} from '../../../node_modules/angular2/testing.d';
```
Comment (dydek) [inline app/app.component.spec.ts]: why with relative path ? and definitions ? hmm looks strange

File: app/app.component.ts [hunk 3]
```diff
@@ -39,6 +40,54 @@ import {About} from './sections/about/about.component';
   {
     path: '/about/:slug', component: About, name: 'AboutSubsection'
   },
+  {
+    path: '/Contact/', component: About, name: 'Contact',
```
Comment (dydek) [inline app/app.component.ts]: I would stick with one Url format with small letter at the beginning

File: app/footer/footer.css [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+.social {
+  margin-top: 10px;
+}
+.social li {
+  float: left;
+  margin-left: 20px;
+}
+.social a {
+  font-size: 20px;
+}
```
Comment (dydek) [inline app/footer/footer.css]: new line at the end :)

File: app/footer/footer.css [hunk 5]
```diff
@@ -0,0 +1,69 @@
+ul {
```
Comment (dydek) [inline app/footer/footer.css]: why footer is in the app folder ?

File: app/footer/footer.html [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+          <li>
+            <a href="">
+              <i class="fa fa-google-plus-square"></i>
+            </a>
+          </li>
+        </ul>
+      </div>
+    </div>
+  </div>
+</footer>
```
Comment (dydek) [inline app/footer/footer.html]: new line

File: app/footer/footer.ts [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+    NgFor,
+    Collapse,
+    DROPDOWN_DIRECTIVES,
+    ROUTER_DIRECTIVES
+  ],
+  styles: [<string> require('./footer.css')]
+
+})
+export class Footer {
+  private menuFooter:Array<any> = menuFooter;
```
Comment (dydek) [inline app/footer/footer.ts]: it's not necessary to define type explicit here

File: app/header/header.css [hunk 8]
```diff
@@ -0,0 +1,188 @@
...
+  border-bottom: 1px solid #edf0f5;
+  background-color: #fff;
+  min-height: 67px;
+}
+.nav > li:focus > a, .nav > li:hover > a {
+  background: #ffc300;
+  color: #fff;
+  border-radius: 2px;
+}
+.navbar-nav > li {
```
Comment (dydek) [inline app/header/header.css]: could we use post-css nested style to have everything more compact and easier to maintain  ? @rychkog  what do you think ?

File: app/footer/footer.css [hunk 9]
```diff
@@ -0,0 +1,69 @@
...
+}
+.block-subscribe form {
+  margin-top: 10px;
+}
+.block-subscribe .text {
+  width: 100%;
+}
+.block-subscribe p {
+  font-size: 20px;
+  color: #767d86;
```
Comment (dydek) [inline app/footer/footer.css]: imo colors should be variables

Comment (natali-abyss) [inline app/footer/footer.css]: yep i'll do it later

File: app/header/header.css (hunk 8 above)
Comment (natali-abyss) [inline app/header/header.css]: now we use stylus https://github.com/stylus/stylus, i removed this file

**Comment by valorkin** [comment]
@natali-abyss @rychkog don't forget to squash commits!

**Pull request merged**